src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -161,17 +161,39 @@
         
 		// keep track of ourselves and whether both server updates succeed
         var self = this;
-        var countSuccesses = 0;
-
+        var callbacksWaiting = 4;
+        
+        // first need to merge any households (Accounts only) before we save contacts
+        // so we avoid deleting a household if that contact was the last one in the hh.
+        var listHHMerge = component.get('v.listHHMerge');
+        var hh = component.get('v.hh');
+        if (listHHMerge != null && listHHMerge.length > 0) {        
+            var action = component.get("c.mergeHouseholds");
+            action.setParams({ hhWinner : hh, listHHMerge : listHHMerge });
+            action.setCallback(this, function(response) {
```

Comment:
At this point you have an identical callback function defined 4 times within this method.  Can you extract it out into its own definition (being careful to maintain closure of the `callbacksWaiting` variable) and just reference it in each `setCallback()` call?